### PR TITLE
Howto specify splits documentation page

### PR DESCRIPTION
**Pull request by pierrot0**

### CODE REVIEW

File: docs/howto/specify-splits.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+The Croissant format allows for the data to be split arbitrarily into one or multiple splits, which for example allows dataset consumers to load a specific split.
+
+## The split field
+
+The splits definition is done at the `RecordSet` level, with a dedicated field which can be named "split" or anything else, but must have the semantic data type [`wd:Q3985153`](https://www.wikidata.org/wiki/Q3985153). That field data type is generally `sc:Text`, but could theoritically come as other data types, `sc:Integer` for example.
+
+The following artificial example defines the split(s) based on a CSV column:
+
+TODO: example snippet, also with `wd:` context declaration.
```
Comment (pierrot0) [inline docs/howto/specify-splits.md:13]: Note for reviewers: I'll need to wait for https://github.com/mlcommons/datasets_format/pull/65 to be merged before I can use the example to dosplay a code snippet.

File: docs/howto/specify-splits.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Often, data from different splits come packaged into different files, which enable dataset consumers to only download the needed data. In such cases, the split definition is still done at the `RecordSet` level, but can refer to data file names or path.
+
+The following example is extracted from the [COCO2014 dataset Croissant config](https://github.com/mlcommons/datasets_format/blob/main/datasets/coco2014/metadata.json):
+
+<https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#L173-L187>
```
Comment (pierrot0) [inline docs/howto/specify-splits.md:19]: Code snippets should render like https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-a-permanent-link-to-a-code-snippet, but I don't know how to preview that...

File: docs/howto/specify-splits.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+The `split_enums` `RecordSet` is declared as a `name` (on which join is done), and the semantic value: a `sc:URL` of type [`wd:Q3985153`](https://www.wikidata.org/wiki/Q3985153):
+
+https://github.com/mlcommons/datasets_format/blob/2e05069f02a93e6154137aa267e69bd1271a95b0/datasets/coco2014/metadata.json#L128-L153
+
+and the data describing those splits actually come from a [CSV file](https://github.com/mlcommons/datasets_format/blob/main/datasets/coco2014/data/splits.csv):
+
+https://github.com/mlcommons/datasets_format/blob/83dd4919ea32319703a0fa23ed013719548b4c03/datasets/coco2014/data/splits.csv#L1
+
+As you can see, here the split URLs are from the [known supported types](#known-supported-data-types) (under `wd:Q3985153`):
+ - https://mlcommons.org/definitions/training_split
```
Comment (josvandervelde) [inline docs/howto/specify-splits.md:42]: These pages do not exist yet. Should we add a "under construction" remark?

**Review by josvandervelde — APPROVED** [review]
Looks good!

File: docs/howto/specify-splits.md (hunk 3 above)
Comment (pierrot0) [reply docs/howto/specify-splits.md:45]: Absolutely, added a warning.

**Pull request merged**